pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -507,6 +518,7 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 			Requested:    1,
 			Used:         int64(len(nodeInfo.Pods)),
 			Capacity:     int64(allowedPodNumber),
+			Unresolvable: false,
```

Comment:
You don't have to put `Unresolvable: false` (because it's false when unset)

Reply:
Fixed